bigbluebutton-html5/imports/ui/components/logout-confirmation/component.jsx

```diff
@@ -1,7 +1,12 @@
 import React, { Component } from 'react';
 import { withRouter } from 'react-router';
 import { defineMessages, injectIntl } from 'react-intl';
+import { makeCall } from '/imports/ui/services/api';
+import Button from '/imports/ui/components/button/component';
 import Modal from '/imports/ui/components/modal/fullscreen/component';
+import Auth from '/imports/ui/services/auth';
+import userListService from '../user-list/service';
```

Comment:
Hmm..
I wonder if we shouldn't pass the methods down through the createContainer in this case. No reactive data here, maybe we need to have a container (just to follow our convention), but then just import the method and pass it to the component directly.